src/main/puzzle/PuzzleHud.tscn

```diff
@@ -0,0 +1,99 @@
+[gd_scene load_steps=9 format=2]
```

Comment:
This is worth mentioning in the commit comment; 'extracted PuzzleHud.tscn from Puzzle.tscn'

Reply:
Commit comment now mentions `Extracted PuzzleHud.tscn from Puzzle.tscn`